base/code/src/components/Footer.tsx

```diff
@@ -1,15 +1,5 @@
 import * as React from 'react';
 
-export class Footer extends React.Component<{
-  className?: string;
-}> {
-  public render() {
-    const { className } = this.props;
-
-    return (
-      <div className={className}>
-        Made with ❤️by Infinum
-      </div>
-    );
-  }
-}
+export const Footer: React.FC<{ className?: string }> = ({ className }) => (
```

Comment:
I would separate the prop types into an interface

```suggestion
interface IFooterProps {
  className?: string;
}

export const Footer: React.FC<IFooterProps> = ({ className }) => (
```